packages/shared/src/styles/custom.ts

```diff
@@ -8,3 +8,8 @@ export const devcardBorder = `color-mix(in srgb, ${colors.salt[90]}, transparent
 
 export const bookmarkProviderListBg =
   'linear-gradient(180deg, var(--theme-actions-bookmark-float) 0%, color-mix(in srgb, var(--theme-accent-bun-bolder), transparent 100%) 100%)';
+
+export const briefCardBorder = '1px solid #EFD5C8';
+
+export const briefCardBg =
```

Comment:
Fine for now, but just keen an eye out or maybe already raise with design if they want to re-use this color more it should be considered in our guidelines.

I forsee them using it more for AI related items, so maybe good to start preparing them for it.